### PR TITLE
USWDS-Site - Implementations: Add MetroStar Comet to USWDS implementations

### LINKED ISSUE #2146: Add new React Metrostar Implementation to Implementations page
Framework: React
MetroStar (https://github.com/MetroStar/comet)
USWDS version: 3.5.0
Notes: React with TypeScript implementation of USWDS, providing additional modular support for custom components, data visualization, and command line tools. Sample starter app included that is focused on accelerating and scaling design across the enterprise.

### PR DESCRIPTION
**Pull request by jbouder**
# Summary

- Added MetroStar Comet Component Library to USWDS Implementations list

## Related issue

Closes #2146

### CODE REVIEW

File: _data/implementations.yml [hunk 1]
```diff
@@ -137,6 +137,16 @@
...
 
+- name: Comet
+  distribution: React
+  id: comet-uswds
+  url: https://github.com/MetroStar/comet
+  author:
+    name: MetroStar
+    url: https://github.com/MetroStar
+  version: 3.5.0
+  notes: "React with TypeScript implementation of USWDS, providing additional modular support for custom components, data visualization, and command line tools. Sample starter app included that is focused on accelerating and scaling design across the enterprise."
```
Comment (amyleadem) [inline _data/implementations.yml:148]: Added a couple of suggestions to help improve clarity. 

```suggestion
  notes: "A React with TypeScript component library based on USWDS 3.0. This library provides modular support for custom components, data visualization, and command line tools. Comet also includes a sample starter app that is focused on accelerating and scaling design across the enterprise."
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
@jboulder Thanks for submitting this PR! I have added a comment with some suggestions to add clarity to the notes. Will you take a look and confirm that everything still makes sense?

Additionally, would you be able to update the [implementations changelog file](https://github.com/uswds/uswds-site/blob/main/_data/changelogs/docs-implementations.yml) with a changelog entry so that users will know when this was added? The data items should follow the same structure as previous entries in that file, but with info specific to this PR. Please reach out if you have any questions or would prefer that we create that entry for you.

**Review by mejiaj — CHANGES_REQUESTED** [review]
LGTM, but requires a new changelog item in [changelogs/docs-implementations.yml](https://github.com/jbouder/uswds-site/blob/f61b03dc3a1dfcb3a36dc9534e53ee5d45fd517f/_data/changelogs/docs-implementations.yml#L4).

**Example**

```yml
  - date: 2023-06-27
    summary: Added entry for MetroStar Comet implementation.
    affectsGuidance: true
    githubPr: 2152
    githubRepo: uswds-site
```

File: _data/implementations.yml (hunk 1 above)
Comment (jbouder) [reply _data/implementations.yml]: Updated as requested. Thank you!

**Comment by jbouder** [comment]
@mejiaj @amyleadem , just wanted to see if there were any updates on this. All should be good to go.

File: _data/changelogs/docs-implementations.yml [hunk 2]
```diff
@@ -2,6 +2,18 @@ title: Implementations
 type: documentation
 changelogURL:
 items:
+  - date: 2023-06-27
```
Comment (amyleadem) [inline _data/changelogs/docs-implementations.yml:5]: > **Note**
We should update the date right before merge.

**Review by amyleadem — APPROVED** [review]
This is looking good! Added a note to the team that we should update the changelog date before merge.

**Review by mahoneycm — APPROVED** [review]
This is great! And the component previews are looking great on the newest version of Storybook! 👍

**Review by mejiaj — APPROVED** [review]
Approved, created this internal PR #2167 to make final changes before merge.

**Comment by jbouder** [comment]
Thanks! Let me know if there is anything more I need to do.

**Comment by mejiaj** [comment]
@jbouder it's live, thanks for your contribution!

**Comment by jbouder** [comment]
@mejiaj , thank you so much!

**Pull request merged**